src/Games/NexusMods.Games.RedEngine/RedModDeployTool.cs

```diff
@@ -34,9 +26,19 @@ public async Task Execute(Loadout.ReadOnly loadout, CancellationToken cancellati
         if (!deployFolder.DirectoryExists())
             deployFolder.CreateDirectory();
 
-        await _processFactory.ExecuteAsync(Cli.Wrap(exe.ToString())
-            .WithArguments("deploy")
-            .WithWorkingDirectory(exe.Parent.ToString()), cancellationToken: cancellationToken);
+        var fs = FileSystem.Shared;
+        if (fs.OS.IsWindows)
+        {
+            var command = Cli.Wrap(exe.ToString())
+                .WithArguments("deploy")
+                .WithWorkingDirectory(exe.Parent.ToString());
+            await _toolRunner.ExecuteAsync(loadout, command, true, cancellationToken);
+        }
+        else
+        {
+            var batchPath = fs.GetKnownPath(KnownPath.EntryDirectory).Combine("Resources/Cyberpunk2077/deploy_redmod.bat");
```

Comment:
This won't work. I'd suggest making the bat file an embedded resource and creating the file at runtime in the temp directory or similar.

Reply:
If I recall correctly, I brought this up at standup.

Generating a .bat and extracting to disk was the thing I originally wanted to do.

However the people there requested I don't do that and make it a static file that ships with the App pre-extracted instead.

So I'm rather confused, why am I being asked here to do what I originally intended to do? This shouldn't be a case of something like `Working Directory` vs `$OWD` for AppImage.

There is the fact I didn't make the `.bat` file get copied on Linux only, but that's a separate fixup.

Reply:
Did you test this with an AppImage? I want to avoid using `CopyToOutputDirectory` if possible, as it can make packaging difficult.

Reply:
Good point. Let me do that after I fixup CI

Reply:
Took me a bit to respond. I just downloaded Cyberpunk 3 times, because Steam would give me errors after downloading the entire 80GB.

Works fine on AppImage, just need it to not copy to output on Windows.

Reply:
I made the file not copy on Windows, however it depends on the Host OS the App was built in.
Details in the commit contents.

Reply:
I still think we should go with the embedded route. `CopyToOutputDirectory` is just a hassle.

Reply:
> Putting it as an embedded resource will make it embed in Windows too

How? What you have currently only copies for Linux. `EmbeddedResource` can have a condition as well.

Reply:
Ah, I thought you wanted me to use the standard machinery here because the `.targets` file was unsightly to see.
If it's embedding via the targets file, then that would work.